src/backend/metal/src/conversions.rs

```diff
@@ -132,6 +153,661 @@ impl PrivateCapabilities {
             }
         }
     }
+
+    pub fn map_format_properties(&self, format: MTLPixelFormat) -> Properties {
+        use hal::format::ImageFeature as If;
+        use metal::MTLPixelFormat::*;
+
+        let defaults = Properties {
+            linear_tiling: If::empty(),
+            optimal_tiling: If::SAMPLED | If::BLIT_SRC | If::BLIT_DST,
+            buffer_features: BufferFeature::all(),
+        };
+
+        match format {
+            A8Unorm => Properties {
+                optimal_tiling: If::SAMPLED_LINEAR,
```

Comment:
I believe what you actually want (here and below) is:
```rust
optimal_tiling: defaults.optimal_tiling
  | If::EXTRA_1
  | If::EXTRA_2,
```

Reply:
Should be addressed.